flag/service/controller/controller.go

```diff
@@ -1,6 +1,7 @@
 package controller
 
 type Controller struct {
-	BackOffDuration string
-	ResyncPeriod    string
+	BackOffDuration          string
+	FrameworkBackOffDuration string
```

Comment:
What is the first backoff duration for? The second is obviously for the framework. I would like to see a consistent informative naming.